ci/Debian11Container.dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:lts-bullseye
+WORKDIR /
+SHELL ["/bin/bash", "-c"]
+
+# Install emscripten
+RUN git clone https://github.com/emscripten-core/emsdk.git \
```

Comment:
emscripted for compiling C dependencies

Reply:
Good idea, I have expanded the comments inside `build-linux.dockerfile`  in commit 749963442fd0f2ecd6408daaf32f4f82ce2376a7.